pkg/controller/controller.go

```diff
@@ -632,24 +632,36 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 			cont.opflexDeviceDeleted(dn)
 		})
 	go cont.apicConn.Run(stopCh)
-	go cont.enableGlobalSync(stopCh)
 }
 
 func (cont *AciController) enableGlobalSync(stopCh <-chan struct{}) {
```

Comment:
This should have ideally been called snatGlobalInfoSync() unless we have plans to expand syncing other resources also in this function. (This can be taken up in a follow-up patch.)